cmd/charmd/config.yaml

```diff
@@ -1,3 +1,4 @@
+#terms-location: localhost:8081
```

Comment:
Nitpick: I'd remove this, or add a comment like `# uncomment to test with a terms service running locally`